README.md

```diff
@@ -27,16 +24,21 @@ go build
 **_NOTE:_** Kubernetes contexts can be shown using `kubectl config get-contexts` and set with `kubectl config set-context`.
```

Comment:
Is having `kubectl` installed a pre-requisite? Could you give a worked example of how to get to the point where the tool can be run. Imagine someone is not that familiar with Kubernetes.

Key information is where the tool should be run, copied onto the target machine, run on your machine with kubectl` installed?